### PR TITLE
install python dependencies by pip

### PR DESCRIPTION
**Pull request by ipa-rwu**
pip install -m requirement.txt
if rosdep resolve <pkg> return error

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# araig_test_stack
```
Comment (ipa-kut) [inline README.md:1]: Please Insert:

# ARAIG Test Stack
This stack contains the GUI, calculators, interpreters and runners required to implement any ARAIG test. 

The actual launch files per robot will be stored in [araig_robot_packages](https://github.com/ipa320/araig_robot_packages)

Benchmarking for these components per test will be stored in [araig_benchmarks](https://github.com/ipa320/araig_benchmarks)

## Requirements

Install python dependencies not supported by rosdep with:
`pip install -m requirement.txt`

File: requirement.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+multipledispatch
+asyncio ; python_version == '3.6'
```
Comment (ipa-kut) [inline requirement.txt:2]: Please add scipy as well

**Review by ipa-kut — CHANGES_REQUESTED** [review]
Please update the readme and add scipy

**Comment by ipa-rwu** [comment]
rosdep support scipy
you can check by 
`rosdep resolve python-scipy`

**Comment by ipa-kut** [comment]
> rosdep support scipy
> you can check by
> `rosdep resolve python-scipy`

Yes but `rosdep install python-scipy` does not work for me. Does it work for you?

**Review by ipa-kut — APPROVED** [review]

**Pull request merged**